src/button/__tests__/button.test.js

```diff
@@ -50,6 +50,18 @@ describe('Button Component', () => {
     expect(component.find(LoadingSpinner)).toExist();
   });
 
+  test('onClick called with event', () => {
+    const onClick = jest.fn();
+    const component = mount(<Button onClick={onClick} />);
+    component.simulate('click', {
+      target: 'foo',
+    });
+    expect(onClick).toHaveBeenCalledTimes(1);
+    expect(onClick.mock.calls[0][0]).toMatchObject({
```

Comment:
in this case we can use `toEqual` as nothing is mixed in

Reply:
Tried that, React/enzyme still wraps the simulated event in a wrapper object (SyntheticEvent?) so they don't match.